website/for_teachers.py

```diff
@@ -1123,26 +1145,71 @@ def parse_preview_adventure(self):
             return gettext("something_went_wrong_keyword_parsing"), 400
         return {"code": code}, 200
 
-    @route("/create-adventure", methods=["POST"])
+    def add_adventure_to_class_level(self, user, class_id, adventure_id, level, remove):
```

Comment:
just to make it clear what you're removing.

```suggestion
    def add_adventure_to_class_level(self, user, class_id, adventure_id, level, should_remove_adv):
```

Reply:
would `remove_adv` also be fine? to make it shorter

Reply:
Sure!